Update user-namespaces.md (#45702)
kubernetes/website: content/en/docs/tasks/configure-pod-container/user-namespaces.md

> 1. Attach to the container and run `readlink /proc/self/ns/user`:

Reviewer: ```suggestion
1. Exec into the Pod and run `readlink /proc/self/ns/user`:
```

   ```shell

>    kubectl exec -it userns -- bash

Reviewer 2: It looks like `kubectl attach` behavior has changed slightly since this was added. Running `exec` is probably fine, but the instructions should be updated to not call out "attach"-ing to the container.

And if you're willing to do it, I think it could be a little confusing that this bullet item then switches back to running a command on the host with no detail. On line 94, I think it would make sense to add a single line saying something like:

```txt
Type `exit` and press enter to disconnect from the `userns` pod.
```

Then start another bullet item for the instructions of running locally on the host:

```txt
1. Compare the ns user on the host

   From a shell on the host, run the same command.

   The output must be different...
```

Author: ok,hahah

Reviewer: > the instructions should be updated to not call out "attach"-ing to the container.

I think that needs a slightly broader change.

Reviewer: Would this work:
```bash
kubectl attach userns -c shell -i -t /bin/sh
```
?
